scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -156,6 +91,84 @@ class ThemeAggregator {
     return assets;
   }
 
+  /**
+   * Triggers aggregation and generation for a given theme.
+   * @param {string} themeName - The theme to aggregate.
+   * @param {Object} options - The aggregation options.
+   * @param {boolean} defaultFlag - Whether the theme to be generated is a root or scope theme. Guards against default theme and scope theme being equivalent.
+   * @returns {array} - An array of file names.
+   */
+  static triggerAggregationAndGeneration(themeName, options = {}, defaultFlag) {
+    if (!themeName) return null;
+    Logger.log(`Aggregating ${themeName} files...`, { LOG_CONTEXT });
+    const { theme, scoped = [] } = options;
+    const isRoot = themeName === theme && defaultFlag;
+    const isScoped = scoped.indexOf(themeName) > -1;
+    const themeScope = { isRoot, isScoped };
```

Comment:
can we condense this to 
```
const themeScope = { isScoped: .isScoped && !themeScope.isRoot };
```
It looks like this check is used multiple times throughout

Reply:
Flattened this to a `isRoot` flag. https://github.com/cerner/terra-toolkit/pull/376#discussion_r367621604